### DISCUSSION TITLE
This repo is getting big, some proposals for maintanance :

### DISCUSSION THREAD

**Discussion by altryne** [hlky/stable-diffusion-webui#223] [General]
Can we add the following things to the repo pelase? 
- Releases  : Let's cut a release when things are "stable" so folks can download the latest release instead of the tip of the master which may contain bugs. Releases would also be a great place to show the new features added
- Wiki - Would be cool to move the install instructions from here https://rentry.org/GUItard to a wiki that folks can contribute to. 
- Support - @hlky I think you're doing amazing work, I think you should set up a supporters link for this repo
- Contribution guidelines
- Bug / Feature request templates for issues
- Github community - there seems to be many folks using this, some on discords, some on twitter, some on reddit and 4chan. We can coordinate all of their questions using the github community

**Comment by altryne** [comment]
I'm willing to help with maintenance @hlky

**Comment by AbdBarho** [comment]
I'm willing to help as well!

Just a small comment regarding releases:

All important changes in this repo are synced against https://github.com/hlky/stable-diffusion so I would assume the master branch there is the stable one, not here.

**Comment by hlky** [comment]
@altryne 

>stable release

We can do this whenever, but I would like to get img2img optimized working and there's probably other stuff to add before a true stable release

>wiki

I can have a look at doing this, there's some extra info on the readme for both repos now

>supporters link

There are crypto addresses on my profile, but I don't like the idea of plastering sponsorship all over the repo itself

>Contribution guidelines

This would be useful to try and keep things standardized 

>Bug / Feature request templates for issues

This has been done by someone, thanks for the help.

>We can coordinate all of their questions using the github community

Is this the discussions feature, or what do you mean?

**Comment by eremtsov** [comment]
I have no idea who supports https://rentry.org/GUItard and where would be the proper place to report this, but I figured this issue is the closest thing. 

As of now, step 2a of the guide is not foolproof. If you just copy the files over you won't achieve anything because the two repos have a different folder structure.

**Comment by hlky** [comment]
Step 2a needs removing, guide users should use the version in the main repo which is considered stable, only unless they know what they are doing or are asked to use the dev version to test a bug fix.

The sync action used to merge changes between repos takes care of the different file structure.

**Comment by eremtsov** [comment]
Thanks, I think it makes sense. But I think people who can help with the development and testing should be able to find it. So maybe move it to the links section.

Another issue I'd like to point out is that prompt weighting is mentioned in the repo readme and the guide, but its usage isn't explained anywhere.

**Comment by hlky** [comment]
I've added a link to this repo from the main repo, unfortunately i dont maintain the guide, there are plans to move the guide content to a wiki on here

Every feature needs properly documenting tbh

**Comment by altryne** [comment]
@hlky I missed these 😓 
Yeah I meant the discussions feature here on github. 
Also, wiki for installation so that guitard thingie will hopefully not confuse folks (and won't have to get updated) 
Thanks for the replies

**Comment by mohsen1** [comment]
As someone who doesn't touch Python often I'm super confused how to install dependencies and launch the web UI

The root README should have the most basic steps to launch this (assuming user already have SD running locally)

**Reply by altryne** [reply]
This repo is for development, please refer to the installation guide here: 
https://github.com/hlky/stable-diffusion/wiki/Installation
